doc/make.jl

```diff
@@ -0,0 +1,144 @@
+# Install dependencies needed to build the documentation.
+ENV["JULIA_PKGDIR"] = joinpath(@__DIR__, "deps")
+Pkg.init()
+write(Pkg.dir("REQUIRE"), "Documenter")
```

Comment:
specifying the versions is a bit more visible. can do "Documenter 0.8.2 0.8.2+" to fix a specific version